.github/workflows/weekly-resources-clean.yml

```diff
@@ -15,7 +15,7 @@ name: Weekly Resources Cleaner
 
 on:
   schedule:
-    - cron: "0 0 * * SAT" # Run every saturday at midnight
+    - cron: "0 0 */5 * *" # Run every 5 days
```

Comment:
What's this change for?

Reply:
Since we destroy every resources that are created by the integration test framework for 5 days; therefore, it would be better for us to set the schedule in every 5 days thought.

Reply:
This should be weekly - on a weekday e.g. mondays

Reply:
I agree that we could delete the resources weekly thought; however, IMO it should be better to keep it up with past day delete (e.g if we decide to delete EC2 instances that have existed over the past 5 days, then we could delete every instances in every 5 days though.